src/jquery.waitforimages.js

```diff
@@ -209,6 +203,9 @@
 
                 });
 
+                if (typeof img.srcset !== "undefined") {
```

Comment:
It’s safer to check against `"undefined"` for browsers that do not support `srcset`.